src/index.ts

```diff
@@ -133,7 +133,10 @@ function onError(err: AxiosError) {
   config.currentRetryAttempt = config.currentRetryAttempt || 0;
   config.retry =
     config.retry === undefined || config.retry === null ? 3 : config.retry;
-  config.retryDelay = config.retryDelay || 100;
+  config.retryDelay =
```

Comment:
Let's flip this around just a bit and do:
```js
config.retryDelay = typeof config.retryDelay === 'number' ? config.retryDelay : 100;
```

Reply:
I like that, I took the liberty to change other 2 configs with similar condition to this format and added tests to them. Is it okay?